CHANGELOG-zh-CN.md

```diff
@@ -1,5 +1,108 @@
 # OceanBase Developer Center (ODC) CHANGELOG
 
+## 4.2.3 (2023-12-15)
+
+### 功能变化
+
+导入导出
+
+- 支持原生 MySQL 数据源的导入导出
+
+数据库对象管理
+
+- OceanBase MySQL 以及原生 MySQL 模式下支持 GIS 数据类型
+
+数据源
+
+- 允许数据源绑定到项目
+
+命令行窗口
+
+- 允许自定义禁用命令
+
+基于项目的管控协同
+
+- 增加 2 个内置项目角色：安全管理员，参与者
+- 允许用户申请项目权限
+
+### 缺陷修复
+
+SQL 执行
+
+- SQL 执行过程中抛出错误没有国际化
+- 无法在团队空间下执行带有 dblink 的 SQL
+- 安全规则允许的前提下，无法在团队空间中执行 desc 语句
+- OceanBase Oracle 模式下执行 SELECT ... ORDER BY 1 样式的语句时 ORDER BY 会失效
+
+结果集导出
+
+- 任务没有日志打印
+
+PL 对象
+
+- 交互式创建函数过程中，无法通过下拉菜单定义 sys_refcursor 类型的返回值
+- OceanBase MySQL 模式下 PL 参数值没有对单引号转义
+
+DLM
+
+- 数据库连接池过小导致任务执行失败
+
+分区计划
+
+- 在 OceanBase 1.4.79 版本的 MySQL 模式下创建任务失败
+
+SQL-Check
+
+- 无法识别`alter table xxx drop index`语句为 DROP INDEX 语句
+
+外部审批集成
+
+- 无法识别索引集合中的数据的表达式
+- 外部系统返回的 xml 形式的数据在反序列化时会丢失原始 xml 的根 tag
+
+数据脱敏
+
+- 当扫描到重复列时会导致敏感列添加失败
+
+数据源
+
+- 用户离开项目且不属于任何项目和角色时依然可以创建数据源
+- OceanBase MySQL 以及原生 MySQL 模式下数据库会话中"执行时间"栏目为 0
+- OceanBase Oracle 模式下通过会话变量管理功能进行时间输出格式的修改在 SQL 执行窗口不生效 
+
+堡垒机集成
+
+- 不活跃连接没有被清理
+
+回收站
+
+- 无法删除回收站中的特定对象
+
+模拟数据
+
+- 任务占用过多内存
+- 不支持 ZHSGB232 编码
+- OceanBase MySQL 以及原生 MySQL 模式下无法对宽度在 8 以下的 bit 类型生成任务
+- 无法跳过自增的主键列
+- OceanBase MySQL 以及原生 MySQL 模式下 bit 类型宽度显示错误
+
+### 改进
+
+- 增加了在 SQL Check 以及任务预检查场景下的问题定位功能，允许用户将违反的安全规则和具体的 SQL 联系在一起
```

Comment:
it is new feature, all 'added' like should be new feature

Reply:
done